pegasus/sites.v3/code.org/i18n/public/educate/csc.ar-SA.md.partial

```diff
@@ -27,17 +27,19 @@ social:
 
 <link href="/css/generated/csc-module.css" rel="stylesheet">
 
+{{ ai_skinny_banner_2023 }}
+
 # امتدادات علوم الكمبيوتر Computer Science Connections
 
-تربط أحدث مناهجنا الدراسية بين تعلم علوم الكمبيوتر والمواد الأخرى مثل الرياضيات وفنون اللغة والعلوم والدراسات الاجتماعية. من خلال امتداد علوم الكمبيوتر، تستكشف الصفوف الدراسية من رياض الأطفال حتى الصف الثاني عشر موضوعاتها العادية بطرق جديدة ومثيرة!
```

Comment:
1. The sync in should not comit any change to this files.
2. Removing translations back to english content is very sus.